tests/test_pylint.py

```diff
@@ -4,21 +4,33 @@
 from unittest import TestCase, skipUnless
 import subprocess
 
-try:
-    subprocess.call(["pylint", "--version"])
-    HAVE_PYLINT = True
-except OSError:
-    msg = '** Warning: Could not find pylint. Static checks skipped'
+retcall = subprocess.call(["which", "pylint"]) # which returns 1 if no pylint
```

Comment:
You should try an import and catch an `ImportError` if pylint is a suggested recommends. `which` doesn't scale.